MSBuild warnings when building runtime test projects

Reproduction:
```sh
# Any non-merged runtime test should do
runtime> dotnet build src\tests\JIT\HardwareIntrinsics\General\Vector256\Vector256_ro.csproj -c Release
```
Expected result: test builds without warnings.

Actual result: a couple of `MSB4011` warnings are issued:
```cs
Build succeeded.

C:\Users\Accretion\source\dotnet\runtime\artifacts\tests\coreclr\packages\JIT\HardwareIntrinsics\General\Vector256\Vector256_ro\Vector256_ro.csproj.nuget.g.props(17,5):
  warning MSB4011: "C:\Users\Accretion\.nuget\packages\microsoft.net.illink.tasks\7.0.100-1.22308.1\build\Microsoft.NET.ILLink.Tasks.props" cannot be imported again.
  It was already imported at "C:\Users\Accretion\source\dotnet\runtime\artifacts\tests\coreclr\packages\Common\test_dependencies\test_dependencies\test_dependencies.csproj.nuget.g.props (16,5)". This is most likely a build authoring error. This subsequent import will be ignored. [C:\Users\Accretion\source\dotnet\runtime\src\tests\JIT\HardwareIntrinsics\General\Vector256\Vector256_ro.csproj]
C:\Users\Accretion\source\dotnet\runtime\artifacts\tests\coreclr\packages\JIT\HardwareIntrinsics\General\Vector256\Vector256_ro\Vector256_ro.csproj.nuget.g.props(18,5):
  warning MSB4011: "C:\Users\Accretion\.nuget\packages\microsoft.net.compilers.toolset\4.3.0-3.22281.14\build\Microsoft.Net.Compilers.Toolset.props" cannot be imported again.
  It was already imported at "C:\Users\Accretion\source\dotnet\runtime\artifacts\tests\coreclr\packages\Common\test_dependencies\test_dependencies\test_dependencies.csproj.nuget.g.props (17,5)". This is most likely a build authoring error. This subsequent import will be ignored. [C:\Users\Accretion\source\dotnet\runtime\src\tests\JIT\HardwareIntrinsics\General\Vector256\Vector256_ro.csproj]
C:\Users\Accretion\source\dotnet\runtime\artifacts\tests\coreclr\packages\JIT\HardwareIntrinsics\General\Vector256\Vector256_ro\Vector256_ro.csproj.nuget.g.props(17,5):
  warning MSB4011: "C:\Users\Accretion\.nuget\packages\microsoft.net.illink.tasks\7.0.100-1.22308.1\build\Microsoft.NET.ILLink.Tasks.props" cannot be imported again.
  It was already imported at "C:\Users\Accretion\source\dotnet\runtime\artifacts\tests\coreclr\packages\Common\test_dependencies\test_dependencies\test_dependencies.csproj.nuget.g.props (16,5)". This is most likely a build authoring error. This subsequent import will be ignored. [C:\Users\Accretion\source\dotnet\runtime\src\tests\JIT\HardwareIntrinsics\General\Vector256\Vector256_ro.csproj]
C:\Users\Accretion\source\dotnet\runtime\artifacts\tests\coreclr\packages\JIT\HardwareIntrinsics\General\Vector256\Vector256_ro\Vector256_ro.csproj.nuget.g.props(18,5):
  warning MSB4011: "C:\Users\Accretion\.nuget\packages\microsoft.net.compilers.toolset\4.3.0-3.22281.14\build\Microsoft.Net.Compilers.Toolset.props" cannot be imported again.
  It was already imported at "C:\Users\Accretion\source\dotnet\runtime\artifacts\tests\coreclr\packages\Common\test_dependencies\test_dependencies\test_dependencies.csproj.nuget.g.props (17,5)". This is most likely a build authoring error. This subsequent import will be ignored. [C:\Users\Accretion\source\dotnet\runtime\src\tests\JIT\HardwareIntrinsics\General\Vector256\Vector256_ro.csproj]
    4 Warning(s)
    0 Error(s)
```
This is a "quality of life" issue; the build still succeeds and everything works.

--- a/src/tests/Directory.Build.props
+++ b/src/tests/Directory.Build.props
@@ -196,6 +196,10 @@
     <BuildAsStandalone Condition="'$(BuildAsStandalone)' == ''">true</BuildAsStandalone>
     <OutputType Condition="$(BuildAsStandalone)">Exe</OutputType>
     <TestFramework>GeneratedRunner</TestFramework>
+
+    <!-- Prevent project-specific NuGet props/targets from clashing with the ones we manually import below. -->
+    <ImportProjectExtensionProps>false</ImportProjectExtensionProps>
+    <ImportProjectExtensionTargets>false</ImportProjectExtensionTargets>
   </PropertyGroup>
 
   <Import Project="$(RepositoryEngineeringDir)testing\tests.props"  Condition="'$(IsTestsCommonProject)' != 'true'" />

